Tasks/Common/AzureHelpers/PrivateFunctions.ps1

```diff
@@ -62,107 +68,217 @@ function Initialize-AzureSubscription {
             $additional['CurrentStorageAccountName'] = $StorageAccount
         }
 
+        Write-Host "##[command]Set-AzureSubscription -SubscriptionName $($Endpoint.Data.SubscriptionName) -SubscriptionId $($Endpoint.Data.SubscriptionId) -Certificate ******** $(Format-Splat $additional)"
         Set-AzureSubscription -SubscriptionName $Endpoint.Data.SubscriptionName -SubscriptionId $Endpoint.Data.SubscriptionId -Certificate $certificate @additional
         Set-CurrentAzureSubscription -SubscriptionId $Endpoint.Data.SubscriptionId
```

Comment:
Per Yas, add storage account when calling Set-CurrentAzureSubscription for certificate.